.github/workflows/land-blocking.yml

```diff
@@ -45,6 +40,24 @@ jobs:
           done
           echo "Build failed after retries"
           exit 1
+  build-and-run-cluster-test:
+    name: Build images and run cluster test
+    needs: build-images
+    runs-on: self-hosted
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup env
+        run: |
+          echo "::set-env name=LIBRA_GIT_REV::$(git rev-parse --short=8 HEAD)"
```

Comment:
This is fine. But personally not a fan of boiler plate because it makes the whole workflow error prone to human errors like updating the env var in one job but miss another.   Wonder if there is a way to set the env var or output in one job and use them in subsequent jobs.

Reply:
I'll try to see if I can improve this before I land

Reply:
I couldn't find any way to share env variables from one job to another.